src/composables/useFlawAffectsModel.ts

```diff
@@ -171,7 +173,7 @@ export function useFlawAffectsModel(flaw: Ref<ZodFlawType>) {
     if (affectCvssToUpsert.value.length) {
       let cvssScoresSavedCount = 0;
       for (const affect of affectCvssToUpsert.value) {
-        // For any new affects that have just been saved, we need to update the affect's
+        // For any new affects that have just been saved, we need to update the affect's uuid
         if (!affect.uuid) {
           const savedAffect = savedAffects.find(matcherForAffect(affect));
 
```

Comment:
I have [a correction to the affect cvss change logic](https://github.com/RedHatProductSecurity/osim/pull/478/commits/05ab08ba8367cd0c8bfae2aba2401ae6f7f98efb)